docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,481 @@
+
+After the transaction queries the parameters approved by channel members, the transaction is then submitted to the ordering service to be added to a block and distributed to the channel. The peers on the channel then validate whether a sufficient number of organizations have approved the chaincode definition. The `peer lifecycle chaincode commit` command will wait for the validations from the peer before returning a response.
+
+You can use the [peer lifecycle chaincode querycommitted](commands/peerlifecycle.html#peer-lifecycle-chaincode-querycommitted) command to confirm that the chaincode definition has been committed to the channel.
+```
+peer lifecycle chaincode querycommitted --channelID mychannel --name fabcar --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+If the chaincode was successful committed to the channel, the `querycommitted` command will return the sequence and version of the chaincode definition:
+```
+Committed chaincode definition for chaincode 'fabcar' on channel 'mychannel':
+Version: 1, Sequence: 1, Endorsement Plugin: escc, Validation Plugin: vscc, Approvals: [Org1MSP: true, Org2MSP: true]
+```
+
+## Invoking the chaincode
+
+After the chaincode definition has been committed to a channel, the chaincode will start on the peers joined to the channel where the the chaincode was installed. The chaincode is then ready to be invoked by client applications. Because we included the ``--init-required`` flag in the chaincode definition, the first transaction needs invoke the Init function to initialize the chaincode. Issue the following command to invoke Init. Note that the invoke command needs to contain the ``--isInit`` flag and target a sufficient number of peers to meet chaincode endorsement policy.
+```
+peer chaincode invoke -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n fabcar --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt --isInit -c '{"function":"initLedger","Args":[]}'
+```
+If the command is successful, you should be able to a response similar to the following:
+```
+2020-02-12 18:22:20.576 EST [chaincodeCmd] chaincodeInvokeOrQuery -> INFO 001 Chaincode invoke successful. result: status:200
+```
+
+We can now use the Fabcar chaincode to create an initial set of cars on the channel ledger.
+```
```

Comment:
So we've been calling it the "blockchain ledger" to this point, which as I've said isn't my favorite term. Now it's the "channel ledger". Not to needlessly continue to strike a fallen filly, but I think "ledger" by itself should suffice.

Reply:
done.